lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -69,6 +73,11 @@ class KubernetesMetadataFilter < Fluent::Filter
     config_param :allow_orphans, :bool, default: true
     config_param :orphaned_namespace_name, :string, default: '.orphaned'
     config_param :orphaned_namespace_id, :string, default: 'orphaned'
+    config_param :checkpoint_enabled, :bool, default: true
+    config_param :checkpoint_interval, :integer, default: 30
+    # This is used to prune older entries that will be written to the sqlite db, the most recent entries(aka the hot LRU cache) will always be in the db.
+    config_param :checkpoint_ttl, :integer, default: 3000
+    config_param :checkpoint_db_path, :string, default: '/var/log/k8s-metadata-checkpoint.db'
```

Comment:
wouldnt a more appropriate path that meets convention be something like `/var/lib/fluent-plugin-kubernetes_metadata_filter/checkpoint.db` or `/var/lib/fluent/k8s-metadata-checkpoint.db`

Reply:
Sure, let me change that to `/var/lib/fluent-plugin-kubernetes_metadata_filter/checkpoint.db`